maze_transformer/training/maze_dataset.py

```diff
@@ -381,16 +361,15 @@ def start_end_distance(maze: SolvedMaze, min_distance: int) -> bool:
     @register_wrap_dataset_filter
     @staticmethod
     def cut_percentile_shortest(
-        dataset: MazeDataset, percentile: float = 0.1
+        dataset: MazeDataset, percentile: float = 10.0
```

Comment:
I absolutely did not realize that `np.percentile` took a percent between 0 and 100. maybe add a comment to the docstring about this, seems like a gotcha?